src/ActionList/Header.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import styled, {css} from 'styled-components'
+import {get} from '../constants'
+
+export interface HeaderProps extends React.ComponentPropsWithoutRef<'div'> {
```

Comment:
Can you add `/** JS Doc */` to exported items?

Reply:
Accidental omission, thanks! I’ll also add https://github.com/gajus/eslint-plugin-jsdoc#eslint-plugin-jsdoc-rules-require-jsdoc as a future reminder.

Reply:
Thanks 💟 

As I read through the rest of the PR, I realized there are no comments whatsoever, so it would be a nice addition!

Reply:
From @smockle in https://github.com/primer/components/pull/1078#discussion_r603310377:
>  I’ll also add https://github.com/gajus/eslint-plugin-jsdoc#eslint-plugin-jsdoc-rules-require-jsdoc as a future reminder.

Actually, I won’t—its [license](https://github.com/gajus/eslint-plugin-jsdoc/blob/master/LICENSE) isn’t auto-recognized as MIT or similar, and I don’t want to seek clarification for something non-essential.

Reply:
Added comments: https://github.com/primer/components/pull/1078/commits/69f1cf0676ec5e277a3e75ccad27bd9a78b1246c 🚀